include/token.h

```diff
@@ -46,4 +46,9 @@ t_token						*new_redir_token(int type, t_token *previous,
 
 void						free_token(t_token *token);
 
-#endif
+void						print_token(t_token *token);
+void						conditional_print(char *string, int start_of_string, int i, t_token **token_list);
```

Comment:
Just a nitpick, **not critically important**: Please make helper functions that aren't used outside of one file `static` and add the prototypes to the file in which the functions are declared instead of adding them to the header file. See 
[free_node.c](https://github.com/dpu-kaiser/minishell/blob/main/src/free_node.c) for example.

Reply:
done